rubin_jupyter_utils/helpers/functions.py

```diff
@@ -201,7 +201,7 @@ def parse_access_token(endpoint=None, tokenfile=None, token=None, timeout=15):
     # Endpoint is constant in an ArgoCD-deployed cluster
     if not endpoint:
         endpoint = "http://gafaelfawr-service.gafaelfawr:8080/auth/analyze"
-    resp = requests.post(endpoint, data={"token": token}, timeout=timeout)
+    resp = requests.post(url=endpoint, data={"token": token}, timeout=timeout)
```

Comment:
I'm just curious why this is necessary, it's a positional argument according to the API docs

Reply:
Yeah, I dunno either, but give it a try as-is on data-dev!